Classes/Utility/AlertAction.swift

```diff
@@ -106,20 +106,24 @@ struct AlertAction {
         return UIAlertAction(title: NSLocalizedString("Discard", comment: ""), style: .destructive, handler: handler)
     }
     
-    static func toggleIssue(_ status: IssueStatus, handler: AlertActionBlock? = nil) -> UIAlertAction {
+    static func toggleIssue(_ status: IssueStatus, issue: Bool, handler: AlertActionBlock? = nil) -> UIAlertAction {
+        let issueType = issue ? "Issue" : "Pull Request"
```

Comment:
Probably safe to wrap these in `NSLocalizedString`?

Reply:
No need as they're used in a NSLocalizedString directly beneath, if these were localized it would cause havoc down the line because you'd have

"Issue" = "**eisiúint**"
"Open **eisiúint**" = "Ceist oscailte" 

Which just over complicates things IMO

Reply:
Good point 👍